jetty-client/src/main/java/org/eclipse/jetty/client/http/HttpReceiverOverHTTP.java

```diff
@@ -252,6 +252,7 @@ private boolean parse()
 
             if (complete)
             {
+                // TODO this can clean a pipelined response after a 1xx
                 if (LOG.isDebugEnabled())
                     LOG.debug("Discarding unexpected content after response: {}", networkBuffer);
                 networkBuffer.clear();
```

Comment:
@sbordet @lorban this is the cause of the UNIX_DOMAIN flake.  A pipelined response can be discarded after a 1xx response.    I'm not sure of the fix.